tutorials/vercel-setup-guide/index.md

```diff
@@ -0,0 +1,254 @@
+
+Click the “Deploy” button to finalise the project’s deployment.
+
+When the project completes deploying, you’ll be redirected to the project deployment’s congratulatory page seen below.
+
+![Screenshot of deployed app success page on Vercel](03-verce-app-deployment-success-page.png)
+
+## 3. Verify the app works
+
+In step 1, you  created and seeded the database with some data. You can view the deployed app by clicking the project’s screenshot at the end of step 2 to view the seeded data and verify that the app is correctly connected to Turso.
+
+![Screenshot of the deployed app preview](04-vercel-deployed-app-preview.png)
+
+Here onwards, Vercel will automate continuous deployments whenever you push to the created git repository’s production branch unless this setting is manually disabled.
+
+## (Alternative) manual deployment
+
+To manually set up your project for deployment on Vercel, without the help of the “Vercel deploy button” in step 2, you can instead use the Vercel dashboard to manually pick your GitHub repo and configure its environment variables.
+
+The following instructions will walk you through this process using the same source repository.
+
+### 1. Fork the repo
+
+Visit [the project on GitHub](https://github.com/turso-extended/app-top-web-frameworks) and fork the repository to your account.
+
+### 2. Add a new project on your Vercel dashboard
+
+Open your Vercel dashboard, select “Add New”, and pick “Project”.
+
+![Screenshot of adding a new project on the vercel dashboard](05-vercel-add-new-project.png)
+
+### 3. Authorise Vercel
+
+If you’ve not authorised Vercel on your GitHub account you’ll be requested to provide such access.
+
+![Screenshot of installing Vercel to a GitHub account](06-vercel-authorising-vercel.png)
+
+Likewise, at the repository level.
+
+![Screenshot of authorising Vercel access to a GitHub repository](07-vercel-authorise-at-repository-level.png)
+
+### 4. Pick your GitHub repository
+
+Next, pick the forked GitHub repository and import it.
+
+![Screenshot of importing a GitHub repository on the Vercel deployment](08-vercel-import-github-repository.png)
+
+### 5. Configure the site settings
+
+Set the site settings for your project on Vercel, including the project’s name, build, and output configurations. (For most frameworks, as is the case here, Vercel auto-detects and populates the default configuration for the project.)
+
+![Screenshot of configuring site settings on Vercel](09-vercel-configuring-site-settings.png)
+
+### 6. Configure the app’s environment variables
+
+The app (Top Web Frameworks) requires two environment variables to facilitate the connection to the database created earlier: `NUXT_TURSO_DB_URL` and `NUXT_TURSO_DB_AUTH_TOKEN`.
+
+Expand the “Environment Variables” section on the site settings page.
+
+These values are required by the [LibSQL TypeScript client](https://github.com/libsql/libsql-client-ts/) SDK to initialise the client and connect to the Turso database.
```

Comment:
American English uses "initialize" with a Z, which is the standard for these docs.